### PR TITLE
[Enhancement][Lake] Add some metrics and logs for compaction jobs

### PR DESCRIPTION
**Pull request by sduzh**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 2 / 50 (04.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionScheduler.java|0|47|00.00%|[106, 123, 129, 131, 132, 133, 151, 152, 154, 155, 156, 157, 166, 252, 253, 344, 345, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 372, 373, 374, 375, 445, 446, 449, 453, 454, 457, 461, 462, 465, 469]|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionManager.java|1|2|50.00%|[106]|
|:large_blue_circle:|com/starrocks/common/Config.java|1|1|100.00%|[]|

**Comment by sduzh** [comment]
run starrocks_be_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 2 / 50 (04.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionScheduler.java|0|47|00.00%|[106, 123, 129, 131, 132, 133, 151, 152, 154, 155, 156, 157, 166, 252, 253, 344, 345, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 372, 373, 374, 375, 445, 446, 449, 453, 454, 457, 461, 462, 465, 469]|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionManager.java|1|2|50.00%|[106]|
|:large_blue_circle:|com/starrocks/common/Config.java|1|1|100.00%|[]|

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

File: fe/fe-core/src/main/java/com/starrocks/common/Config.java [hunk 1]
```diff
@@ -1754,6 +1754,9 @@ public class Config extends ConfigBase {
     @ConfField
     public static int lake_compaction_max_tasks = -1;
 
+    @ConfField(mutable = true)
+    public static boolean lake_verbose_compaction_log = false;
```
Comment (wyb) [inline fe/fe-core/src/main/java/com/starrocks/common/Config.java:1758]: not necessary to add this config, you can set `sys_log_verbose_modules = com.starrocks.lake.compaction` to print lake compaction debug logs.

**Review by wyb — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 45 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionManager.java|0|1|00.00%|[106]|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionScheduler.java|0|44|00.00%|[119, 125, 127, 128, 129, 147, 148, 150, 151, 152, 153, 162, 248, 249, 352, 353, 354, 355, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 441, 442, 445, 449, 450, 453, 457, 458, 461, 465]|

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Review by gengjun-git — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 1 / 45 (02.22%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionManager.java|0|1|00.00%|[106]|
|:large_blue_circle:|com/starrocks/lake/compaction/CompactionScheduler.java|1|44|02.27%|[119, 125, 127, 128, 129, 147, 148, 150, 151, 152, 153, 248, 249, 352, 353, 354, 355, 356, 357, 358, 359, 360, 361, 362, 363, 364, 365, 366, 367, 368, 369, 370, 371, 441, 442, 445, 449, 450, 453, 457, 458, 461, 465]|

**Review by gengjun-git — APPROVED** [review]

**Review by wyb — APPROVED** [review]

**Comment by sduzh** [comment]
run starrocks_admit_test

**Comment by sduzh** [comment]
run starrocks_admit_test

**Comment by sduzh** [comment]
run starrocks_admit_test

**Pull request merged**